tests-integrations/fixtures/eslint-plugin-markdown-nest/test.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
`````suggestion
````md
```json
{
  "quotes": 'value'
}
```
````
`````

json in md in md.

Reply:
Hmm... It seems that eslint v6 parses the contents of ````md as JavaScript 😓.
Only the fixture for v6 testing needs to be separate.

Reply:
Or maybe release as a major version?